[export] Log operator set.

zhxchen17 opened the pull request:
Summary: as title. We want to count the number of total operator calls, and the distinct set of operators in the exported graph.

Test Plan: CI

Differential Revision: D54390298

tugsbayasgalan commented on torch/export/_trace.py:
```diff
@@ -581,6 +581,20 @@ def rewrite_non_persistent_buffers(
+    op_count = 0
+    op_set = set()
+    for m in ep.graph_module.modules():
+        for node in m.graph.nodes:
```
Does this count the top level graph module ops?

zhxchen17 committed:
[export] Log operator set.

Summary: as title. We want to count the number of total operator calls, and the distinct set of operators in the exported graph.

Test Plan: CI

Reviewed By: tugsbayasgalan

Differential Revision: D54390298
